### PR TITLE
docs: fix broken links at translated docs

### PR DESCRIPTION
**Pull request by KunHwanAhn**
The translated documents are using broken relative links.

So I fixed it to use right relative links.

### CODE REVIEW

**Comment by KunHwanAhn** [comment]
https://github.com/anuraghazra/github-readme-stats/pull/770 PR has similar diffs, but my PR includes it. 🤔

File: tests/renderStatsCard.test.js [hunk 1]
```diff
@@ -245,7 +245,7 @@ describe("Test renderStatsCard", () => {
       document.querySelector(
         'g[transform="translate(0, 25)"]>.stagger>.stat.bold',
       ).textContent,
-    ).toBe("总提交 (2020):");
+    ).toBe("总提交 (2021):");
```
Comment (KunHwanAhn) [inline tests/renderStatsCard.test.js:248]: I'm not sure whether we should check this `year`'s number? 🤔 ...

Anyway, it is the reason why test failed, I fixed.

Comment (KunHwanAhn) [reply tests/renderStatsCard.test.js:248]: or we need to write code to check it dynamically? 🤔

**Comment by anuraghazra** [comment]
Thanks for the fix @KunHwanAhn

**Pull request merged**